snmp/tests/test_e2e_core_profiles/test_profile_cisco_ise.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (
+    assert_common_metrics,
+    assert_extend_generic_if,
+    assert_extend_generic_ucd,
+    create_e2e_core_test_config,
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_cisco_ise(dd_agent_check):
+    config = create_e2e_core_test_config('cisco-ise')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:cisco-ise',
+        'snmp_host:cisco-ise.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+    ] + []
+
+    # --- TEST EXTENDED METRICS ---
+    # Examples:
+    assert_extend_generic_if(aggregator, common_tags)
+    assert_extend_generic_ucd(aggregator, common_tags)
+
+    # Extended metrics from `_generic-host-resources-base.yaml`
+    aggregator.assert_metric("snmp.hrSystemUptime", metric_type=aggregator.GAUGE, tags=common_tags)
+    cpu_rows = ['0.0']
+    for cpu_row in cpu_rows:
+        aggregator.assert_metric(
+            'snmp.hrProcessorLoad', metric_type=aggregator.GAUGE, tags=common_tags + ['processorid:' + cpu_row]
+        )
+
+    hr_mem_rows = [
+        ['storagedesc:my-storage-descr'],
+    ]
+    for mem_row in hr_mem_rows:
+        aggregator.assert_metric('snmp.hrStorageSize', metric_type=aggregator.GAUGE, tags=common_tags + mem_row)
+        aggregator.assert_metric('snmp.hrStorageUsed', metric_type=aggregator.GAUGE, tags=common_tags + mem_row)
```

Comment:
why not use the `assert_extend_generic_host_resources_base` since the profile is extending it ?

Reply:
@NouhaManai96 That's because the other metrics in this profile:

```

metrics:
  - MIB: HOST-RESOURCES-MIB
    symbol:
      name: memory.total
      OID: 1.3.6.1.2.1.25.2.3.1.5.1
  - MIB: HOST-RESOURCES-MIB
    symbol:
      name: memory.used
      OID: 1.3.6.1.2.1.25.2.3.1.6.1
  - MIB: HOST-RESOURCES-MIB
    symbol:
      name: cpu.usage
      OID: 1.3.6.1.2.1.25.3.3.1.2.196608
```

share the same OIDs are the ones in `_generic-host-resources-base.yaml` and we need to set precise indexes `.1` and `.196608`.

Reply:
Using the `assert_extend_generic_host_resources_base` won't work, since it expect different indexes.